common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -269,6 +269,19 @@ public void run() {
         }
     }
 
+    private static void logParameters(@NonNull String tag, @NonNull String correlationId, @NonNull Object parameters, @Nullable String publicApiId) {
+        final String TAG = tag + ":" + parameters.getClass().getSimpleName();
+
+        //TODO - conversion of PublicApiId in readable form.
+        Logger.info(TAG, DiagnosticContext.getRequestContext().toJsonString(), "Starting request for correlation id : ####" + correlationId + ", with PublicApiId : " + publicApiId);
```

Comment:
Why so many hashtags here? Looks it would be printed like `Starting request for correlation id : #### <GUID>`

That seems weird...I would expect the value for correlation id to be printed right after the colon

Reply:
Its to make it easier to search the entry and exit point of the request from the command dispatcher using a correlation Id.
The hashtags would only be in 2 statements indicating the same. The exit point is marked with ## followed by a correlation Id.
This can be altered to have less number # or any character or remove them completely, but then you would have to either type the complete stuff like
`Starting request for correlation id : <GUID>` or if you choose to search using a substring of this then probably would have to manually check all the hits to find the right one.

Reply:
I made some changes here @shahzaibj, ## for start of request and ** to specify the end.